src/applications/disability-benefits/2346/tests/mdot.cypress.spec.js

```diff
@@ -1,3 +1,4 @@
+/* disabling old test, soon to be removed
```

Comment:
Disabling this test from the feedback received from the octo-health-engineers:

https://dsva.slack.com/archives/CBU0KDSB1/p1719940947447659?thread_ts=1719940065.431429&cid=CBU0KDSB1